scripts/extensions/pool_update.precheck

```diff
@@ -125,22 +124,26 @@ def execute_precheck(session, control_package, yum_conf_file, update_precheck_fi
                         prerequisite_updates += prerequisite + ' '
                     raise PrerequisiteMissing(prerequisite_updates.rstrip())
                 else:
-                    raise PrecheckFailure(precheckfailure_msg)
+                    raise PrecheckFailure(errmsg)
             else:
-                raise PrecheckFailure(precheckfailure_msg)
+                raise PrecheckFailure(errmsg)
         else:
-            raise PrecheckFailure()
+            raise PrecheckFailure(output)
     else:
         if not os.path.isfile(update_precheck_file):
```

Comment:
If invert this test to `if os.path.isfile(...)` and remove the `return 'ok'` ...